### PR TITLE
Use Typesafe config

### PR DESCRIPTION
**Pull request by ejayaraman**

### CODE REVIEW

File: src/main/g8/src/test/resources/application.conf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    payments-frontend {
+      port = 9050
+      productionRoute = "/pay-online"
+    }
+  }
+}
+
+qa {
+  services {
+    host: "https://www.qa.placeholder.host"
```
Comment (konradwudkowski) [inline src/main/g8/src/test/resources/application.conf]: not essential but hardcoding qa link forces acceptance tests to be private, typesafe config gives you a way to override by using environment variable: https://github.com/lightbend/config#optional-system-or-env-variable-overrides

at some point you may consider it

File: src/main/g8/src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala [hunk 2]
```diff
@@ -97,7 +81,7 @@ trait BasePage extends Matchers {
   def verifyInputUsingElementId(elementId: String, expectedValue: String)= findById(elementId).getAttribute("value") shouldBe expectedValue
 
   def verifyHyperlink(linkText: String) = findByCSS("a[title*=\"" + linkText + "\"]")
-  def verifyHyperlinkTarget(id: String, target: String) = Assert.assertEquals(findById(id).getAttribute("href"), basePageUrl+"/"+target)
+//  def verifyHyperlinkTarget(id: String, target: String) = Assert.assertEquals(findById(id).getAttribute("href"), basePageUrl+"/"+target)
```
Comment (konradwudkowski) [inline src/main/g8/src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala]: comment needed?

File: src/main/g8/src/test/resources/application.conf (hunk 1 above)
Comment (ejayaraman) [reply src/main/g8/src/test/resources/application.conf]: Updated to use environment variable for QA and External Test.

File: src/main/g8/src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala (hunk 2 above)
Comment (ejayaraman) [reply src/main/g8/src/test/scala/uk/gov/hmrc/test/ui/pages/BasePage.scala]: For now yes. We are going to review the usage of these methods in a different ticket. So we will remove this method if required at that point of time.

**Pull request merged**